static/js/components/Dashboard.js

```diff
@@ -14,6 +14,9 @@ class Dashboard extends React.Component {
         </div>
         <div className="card-name">
           { SETTINGS.name }
+          <div className="card-student-id">
+            { profile.student_id }
```

Comment:
Can you also update `USER_PROFILE_RESPONSE` in `constants.js` to add this new field?